### PR TITLE
updating devhub signout view

### LINKED ISSUE #8633: Update Dev Hub - signed out view
Here are the changes for our regularly scheduled Dev Hub (signed out view) updates. 

**Current view:** 
2018 06 22 dev hub - signed out

**Updates**
Remove all text from "Help us improve the developer experience" box. 

Replace with: 

- Headline: Resources for a successful launch
- Copy: You’ve built your extension and are ready to publish it. What comes next? Check out our tips for creating an appealing listing page, getting your extension featured on addons.mozilla.org, finding answers to development questions, and staying up-to-date with news and events relevant to extension developers. 
- CTA: View Resources for Publishers

### PR DESCRIPTION
**Pull request by championshuttler**
Fixes #8633

### CODE REVIEW

File: src/olympia/devhub/templates/devhub/new-landing/components/manage_your_addon.html [hunk 1]
```diff
@@ -6,15 +6,17 @@ <h3 class="DevHub-content-header DevHub-content-header--manage">{{ _('Manage You
...
-        <h3 class="DevHub-callout-header">{{ _('Help us improve the developer experience') }}</h3>
+        <h3 class="DevHub-callout-header">{{ _('Resources for a successful launch') }}</h3>
         <p class="DevHub-callout-copy">
           {% trans %}
-          We’re interested in your feedback about the extension development experience!
-          Take our short survey to let us know what kinds of documentation,
-          APIs, and tools would be most useful to your development work.
+          You’ve built your extension and are ready to publish it. 
+          What comes next? Check out our tips for creating an appealing
+          listing page, getting your extension featured on addons.mozilla.org,
```
Comment (EnTeQuAk) [inline src/olympia/devhub/templates/devhub/new-landing/components/manage_your_addon.html]: I would make `addons.mozilla.org` a link tbh, we tend to do it like this:

1. Define two variables only available in the `{% trans %} block that stands for the link opening and closing: ``{% trans amo_link_open='<a href="https://addons.mozilla.org/">'|safe amo_link_close='</a>' %}``
2. Use that variables in the template tag: ``{{ amo_link_open }}addons.mozilla.org{{ amo_link_close }}``

**Comment by championshuttler** [comment]
Hi @EnTeQuAk , I updated PRs as per your comments. Is it looking right now? 
Sorry but I am in hills area right now and internet is too slow unable to pull docker to run the project locally on my laptop

**Comment by EnTeQuAk** [comment]
no worries, thanks for the update. I'll test this tomorrow morning and merge this if this works as expected.

It looks good to me though.

**Comment by EnTeQuAk** [comment]
I fixed the trans tag and marked the closing `</a>` tag as safe to avoid it from being escaped.

See the attached screenshot for more details
amo_my_addons_new_with_link

Going to merge this once tests are passing.

**Pull request merged**